tests/sentry/snuba/metrics/test_mqb_query_transformer.py

```diff
@@ -0,0 +1,707 @@
+            groupby=[
+                MetricGroupByField("project_id", alias=None),
+                MetricGroupByField("transaction", alias=None),
+            ],
+            limit=Limit(limit=51),
+            offset=Offset(offset=0),
+            include_series=False,
+        ),
+        id="apdex + percentiles test cases",
+    ),
+    pytest.param(
+        Query(
+            match=Entity("generic_metrics_sets"),
+            select=[
+                Function(
+                    function="count_unique",
+                    initializers=None,
+                    parameters=[
+                        Column("s:transactions/user@none"),
+                    ],
+                    alias="count_unique_user",
+                ),
+            ],
+            groupby=[
+                AliasedExpression(
+                    exp=Column(
+                        name="tags[transaction]",
+                        entity=None,
+                        subscriptable="tags",
+                        key="transaction",
+                    ),
+                    alias="transaction",
+                )
+            ],
+            array_join=None,
+            where=[
+                Condition(
+                    lhs=Column(name="timestamp", entity=None, subscriptable=None, key=None),
+                    op=Op.GTE,
+                    rhs=datetime.datetime(2022, 3, 24, 11, 11, 33, 21219, tzinfo=pytz.utc),
+                ),
+                Condition(
+                    lhs=Column(name="timestamp", entity=None, subscriptable=None, key=None),
+                    op=Op.LT,
+                    rhs=datetime.datetime(2022, 6, 22, 11, 11, 33, 21219, tzinfo=pytz.utc),
+                ),
+                Condition(
+                    lhs=Column(name="project_id", entity=None, subscriptable=None, key=None),
+                    op=Op.IN,
+                    rhs=[2],
+                ),
+                Condition(
+                    lhs=Column(name="org_id", entity=None, subscriptable=None, key=None),
+                    op=Op.EQ,
+                    rhs=2,
+                ),
+                Condition(
+                    lhs=Function(
+                        function="tuple",
+                        initializers=None,
+                        parameters=[
+                            Column(
+                                name="tags[transaction]",
+                                entity=None,
+                                subscriptable="tags",
+                                key="transaction",
+                            )
+                        ],
+                        alias=None,
+                    ),
+                    op=Op.IN,
+                    rhs=Function(
+                        function="tuple",
+                        initializers=None,
+                        parameters=[("foo_transaction",), ("bar_transaction",)],
+                        alias=None,
+                    ),
```

Comment:
🤔 Is this from output in our tests? Probably should be
```suggestion
                    rhs=["foo_transaction", "bar_transaction"]
```

Reply:
let me check, maybe I got this one wrong

Reply:
if our tests are doing this lmk; this feels like a bug, or some very specific clickhouse behaviour we worked around (in which case maybe we move it into the metric layer?)

Reply:
agreed